test/e2e-kuttl/quota-forest/10-assert.yaml

```diff
@@ -0,0 +1,13 @@
+# Verify AppWrapper without quota finished successfully
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 60
+---
+apiVersion: mcad.ibm.com/v1beta1
+kind: AppWrapper
+metadata:
```

Comment:
Please add an assertion that the original labels have been preserved and that the "default" labels have been added.

Reply:
Done